packages/server/src/validators/UsersValidator.js

```diff
@@ -0,0 +1,27 @@
+import Models from '../models'
+import BaseValidator from './BaseValidator'
+import { isAddress, toChecksumAddress } from 'web3-utils'
+
+const { UserAddress } = Models
+
+class UsersValidator extends BaseValidator {
+  async validateForCreate({ email, address }) {
+    this._validateEmail(email)
+    await this._validateAddress(address)
+    return this.resetErrors()
+  }
+
+  _validateEmail(email) {
+    if (!email) this.addError({ email: 'An email address must be given' })
```

Comment:
Let's do a bit more of a sanity check here for their email :).

[`joi`](https://www.npmjs.com/package/joi) is quite nice, although it's more intertwined with `hapi`. I'm sure we can find something equivalent for other frameworks that should allow us to declare types just as easily.

Reply:
Yea, we [are using sequelize validations](https://github.com/aragonone/court-backend/pull/73/files#diff-402aeeda5eda2223bd3da4bf15d0c2bbR10) for this, but let's pre-validate here as well